tensorflow/core/kernels/training_ops_gpu.cu.cc

```diff
@@ -637,6 +637,69 @@ struct SparseApplyKerasMomentum<GPUDevice, T, Tindex> {
   }
 };
 
+template <typename T, typename Tindex>
+__global__ __launch_bounds__(1024) void SparseApplyAdadeltaKernel(
+    T* var, T* accum, T* accum_update, const T* lr, const T* rho,
+    const T* epsilon, const T* grad, const Tindex* indices, Tindex param_rows,
+    Tindex updates_size, Tindex indices_size) {
+  Tindex col_size = updates_size / indices_size;
```

Comment:
Do you mean `row_size`?

Reply:
Based on other kernel implementation convention, like `SparseApplyAdagradKernel`, `SparseApplyFtrlKernel` and etc. I named it `col_size`. Do I need to change it?